docs/products/sessions/README.md

```diff
@@ -49,3 +49,15 @@ The equivalent CLI command:
 grid session delete $INTERACTIVE_NODE_ID
 ```
 
+## Share Data Between Sessions
+At this time the only supported way to share data between sessions is to mount the sessions locally and copy files to the local mounts. Establishing SSH connection to your sessions is a prerequisite to this. See these [steps](https://github.com/gridai/grid-docs/blob/doc-182-share-data-between-sessions/docs/products/sessions/how-to-ssh-into-a-session.md) to set up SSH connection with your sessions. **Be sure to logout of the interactive session before attempting the next step**. After that you can do the following to create your session mounts:
```

Comment:
@oojo12 when hyperlinking to an internal link, use relative markdown path so you can update the link here to 
(./how-to-ssh-into-a-session.md) otherwise the link will take them to the page in Github

Reply:
Huh, why do you need to logout of ixsession before this step?

Reply:
@essiequoi making the change now

@nmiculinic because running grid session ssh <session name> logs you into the session. Attempting to perform a session mount from within the session gives you an aws public key error. See the screenshot. Thus the provided instructions are workaround for that by doing it from your local machine.
![error](https://user-images.githubusercontent.com/47154698/153623412-90773852-073a-43b6-a785-542fe3b35d92.png)

Reply:
addressed @essiequoi

Reply:
huh....first things first

> Attempting to perform a session mount from within the session gives you an aws public key error

This is not AWS public key error. It has nothing to do with AWS. It's SSH public key error, since a valid private key isn't present on the session; you'd have to use ssh-agent forwarding for this perhaps, this would require CLI changes to include agent forwarding. What do you think @rusenask ? 

The second error for modprobe fuse is...interesting. Since we're running sessions within a container, can we use a fuse to mount something using it within a non-privileged container? 

https://stackoverflow.com/questions/48402218/fuse-inside-docker  This is interesting, albeit a bit dated (2018). 

This seems to be a more recent find, 2021 https://stackoverflow.com/questions/68709395/how-to-mount-a-fuse-based-filesystem-on-docker-container-running-on-aws  but it still gives a lot of permissions to the session container. CC @filintod for any thoughts.

Reply:
@nmiculinic @filintod how would you like to proceed? If it is additional content I would ask that you just commit it directly instead of placing in the comment something for me to copy + paste. It will be more efficient that way. @essiequoi says this is important for the bug bash.

Reply:
Regarding fuse; currently mounting fusefs within session will fail since fuse device isn't exposed within the container. Though there's:
https://github.com/JasonChenY/fuse-device-plugin
https://github.com/kuberenetes-learning-group/fuse-device-plugin


fuse device plugins for k8s we could investigate and potentially add; allowing users to use fusefs within the sessions (and mount other sessions within original session, to remind we're using sshfs, that is fuse filesystem when doing session mounting)